.gitignore

```diff
@@ -1,6 +1,6 @@
 /cache
 /brick/
 run_dvc.sh
-
+.idea
```

Comment:
By the way, you don't need this. You can set up a global `.gitignore`: <https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/>.